src/base/bittorrent/session.h

```diff
@@ -377,6 +377,8 @@ namespace BitTorrent
         void setAnnounceToAllTiers(bool val);
         int asyncIOThreads() const;
         void setAsyncIOThreads(int num);
+        int checkingMemUsage() const;
+        void setCheckingMemUsage(int num);
```

Comment:
here the parameter is `num` while at definition is `size`, you should use the same name.

Reply:
Ok, fixed